<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelCrum">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crumble/Goo</string>
    </property>
   </widget>
   <widget class="QLabel" name="labeLive">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Live Resin</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCry">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crystals</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDisto">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Distillate</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listCrum">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>171</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listDisto">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>171</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listCry">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>171</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listLive">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>171</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>962</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuYield">
    <property name="title">
     <string>Yield</string>
    </property>
    <widget class="QMenu" name="menuYield_Selected">
     <property name="title">
      <string>Yield Selected</string>
     </property>
     <addaction name="actionCrumble_Goo"/>
     <addaction name="actionLive_Resin"/>
     <addaction name="actionCrystals"/>
     <addaction name="actionDistillate"/>
    </widget>
    <addaction name="menuYield_Selected"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuYield"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionBulk">
   <property name="text">
    <string>Bulk</string>
   </property>
  </action>
  <action name="actionDistolate">
   <property name="text">
    <string>Distolate</string>
   </property>
  </action>
  <action name="actionUnit_Package">
   <property name="text">
    <string>Unit Package</string>
   </property>
  </action>
  <action name="actionCrumble_Goo">
   <property name="text">
    <string>Crumble/Goo</string>
   </property>
  </action>
  <action name="actionLive_Resin">
   <property name="text">
    <string>Live Resin</string>
   </property>
  </action>
  <action name="actionCrystals">
   <property name="text">
    <string>Crystals</string>
   </property>
  </action>
  <action name="actionDistillate">
   <property name="text">
    <string>Distillate</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
